src/components/subscriptions/SubscriptionCard.jsx

```diff
@@ -53,6 +56,11 @@ const SubscriptionCard = ({
   );
 };
 
+SubscriptionCard.defaultProps = {
+  redirectPage: 'subscriptions',
```

Comment:
Should this use the `DEFAULT_REDIRECT_PAGE` constant?

Reply:
YES, yes it should, I undid a bunch of things at one point and it must have gotten lost in the confusion. Good catch.